wpeverest/user-registration
# Enhance - Max uploadable file number setting.

### All Submissions:






### Changes proposed in this Pull Request:
This PR will add a new number field in the general settings section of field options in the form builder.

### How to test the changes in this Pull Request:

1. Navigate to the form builder.
2. Drag and drop the file upload field and observe its field option.
3. A new maximum number of files option will be displayed in the general settings which is a number type field.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Max uploadable file number setting.
